Formula/m/mlc.rb

```diff
@@ -26,7 +25,7 @@ class Mlc < Formula
   def install
     # Explicitly set linker to avoid Cargo defaulting to
     # incorrect or outdated linker (e.g. x86_64-apple-darwin14-clang)
-    ENV["RUSTFLAGS"] = "-C linker=#{ENV.cc}"
+    ENV.append_to_rustflags "-C linker=#{ENV.cc}"
```

Comment:
Looks like these RUSTFLAGS aren't actually making it to `rustc` for some reason:

https://github.com/Homebrew/homebrew-core/actions/runs/16884114006/job/47827335082#step:4:255

Reply:
@carlocab How can it be that these RUSTFLAGS are both breaking builds and not making it through?

Which of these can you reproduce locally?